test/pipeliner/connector.go

```diff
@@ -0,0 +1,59 @@
+package pipeliner
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/amp-labs/connectors/pipeliner"
+	testUtils "github.com/amp-labs/connectors/test/utils"
+	"github.com/amp-labs/connectors/utils"
+)
+
+func GetPipelinerConnector(ctx context.Context, filePath string) *pipeliner.Connector {
+	registry := utils.NewCredentialsRegistry()
+
+	readers := []utils.Reader{
+		&utils.JSONReader{
+			FilePath: filePath,
+			JSONPath: "$.userName",
+			CredKey:  utils.Username,
```

Comment:
username, password are new variables that can appear in the creds.json.
There is a PR https://github.com/amp-labs/connectors/pull/668 that should automatically imply these fields based on catalog information.

Reply:
If these will be used in that PR, can you please move these changes to the other PR? It'll be easier to maintain context in one PR about changes